fate/src/main/java/org/apache/accumulo/fate/util/LoggingRunnable.java

```diff
@@ -34,20 +34,37 @@ public void run() {
     try {
       runnable.run();
     } catch (Throwable t) {
+      boolean runnableError = (t instanceof Error);
       try {
         log.error("Thread \"{}\" died {}", Thread.currentThread().getName(), t.getMessage(), t);
       } catch (Throwable t2) {
-        // maybe the logging system is screwed up OR there is a bug in the exception, like
-        // t.getMessage() throws a NPE
-        System.err.println(
-            "ERROR " + new Date() + " Failed to log message about thread death " + t2.getMessage());
-        t2.printStackTrace();
-
-        // try to print original exception
-        System.err
-            .println("ERROR " + new Date() + " Exception that failed to log : " + t.getMessage());
-        t.printStackTrace();
+        boolean logError = (t2 instanceof Error);
```

Comment:
"logError" isn't a great name, as it could be read as "shouldLogAnError" or "logIsAnError"

How about `loggingCausedAnError` or `errorOnLog` or `logHadError`?

Reply:
That makes sense. I'll make those changes.